### ISSUE TITLE
Move all themes into `prism-themes`?

### ISSUE THREAD

**Issue by RunDevelopment** [PrismJS/prism#2882]
Should we move all themes into `prism-themes`?

Right now, our collection of themes is split into two repositories. This is sorta suboptimal.

---

@LeaVerou @Golmote @mAAdhaTTah

**Comment by LeaVerou** [comment]
Makes sense. It would still require some work to pull out the "official" themes from there to display on the website. It would however be nice to be able to preview all of them somehow.
